app/Functions/Functions.php

```diff
@@ -1659,6 +1719,7 @@ public static function getRelationshipNameFromPath($path, Individual $person1 =
 								default:
 									return I18N::translate('great ×%s aunt/uncle', I18N::number($up - 4));
 							}
+						case 'de':
```

Comment:
Proper number for great-x-nephew/niece (currently translated as "(7-1)× Urgroßnichte", should be "6× Urgroßnichte"). Translations will have to be adjusted once this is accepted.